app/interactors/file_attachments/update_interactor.rb

```diff
@@ -76,14 +76,9 @@ def attachment_params
   def blob_attributes(file_attachment_revision)
     return {} unless attachment_params[:file]
```

Comment:
Do you think this should be a concern of the service?

Reply:
Yeah I think we can move that there.